packages-common/pluggable-widgets-generator/generators/app/templates/typings/customMatchers.d.ts

```diff
@@ -0,0 +1,8 @@
+// note: cannot use ReactElement here, because importing a module from the declaration of a namespace is not allowed.
+declare namespace jasmine {
```

Comment:
I dont think we need it anymore

Reply:
Removed all typings, none of them are needed.